### PR TITLE
KRACOEUS-8741: Demo Qnnr Usage change.

### PR DESCRIPTION
**Pull request by chrisdenne**

### CODE REVIEW

**Comment by kualiCoeusCi** [comment]
Can one of the admins verify this patch?

**Comment by blackcathacker** [comment]
add to whitelist

**Comment by blackcathacker** [comment]
test this please

File: coeus-db/coeus-db-sql/src/main/resources/org/kuali/coeus/coeus-sql/current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8741_000D.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+DELIMITER /
+
+UPDATE QUESTIONNAIRE_USAGE 
+   SET IS_MANDATORY = 'N'
+   WHERE QUESTIONNAIRE_LABEL = 'Generic Demo Questionnaire' AND 
+         MODULE_ITEM_CODE = (SELECT MODULE_CODE FROM COEUS_MODULE WHERE DESCRIPTION = 'Development Proposal')
+/
+
+DELIMITER ;
```
Comment (blackcathacker) [inline coeus-db/coeus-db-sql/src/main/resources/org/kuali/coeus/coeus-sql/current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8741_000D.sql]: Can you add a newline at the end of this just to make script testing easier?

Comment (leoherbie) [inline coeus-db/coeus-db-sql/src/main/resources/org/kuali/coeus/coeus-sql/current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8741_000D.sql]: I'll merge and then do it

**Pull request merged**